### PR TITLE
About us page

### LINKED ISSUE #21: About us page banner
Complete the gradient swoosh with "read more" CTA on the "About Us" page.
- center gradient backgrounds for narrow views, but set a `min-width` on each that looks right
- Use `scrollIntoView` on CTA to jump to the "our mission" section
- Wrap person icon below the content for narrow views
About us page header mockup

### PR DESCRIPTION
**Pull request by ghost**
Resolves #21 

This PR hits basically every part of the new "about us" page. Should have been broken out into smaller PRs... but in the interest of time, I'm not able to divide this up. Happy reviewing!

### CODE REVIEW

File: src/components/about-us/HeaderSwoosh.svelte [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+</style>
+
+<svg
+  width="1440"
+  height="700"
+  viewBox="0 0 1440 832"
+  fill="var(--primary)"
+  preserveAspectRatio="none"
+  xmlns="http://www.w3.org/2000/svg">
+  <path
```
Comment (thejameswang) [inline src/components/about-us/HeaderSwoosh.svelte:20]: Maybe this is all Svelte intuition, but i would never be able to do anything with revising this. This all needs some more documentation, but I understand the time constraint

File: src/components/about-us/MilestonesSection.svelte [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    width: 60px;
+    height: 60px;
+  }
+</style>
+
+<section>
+  <h2>Milestones</h2>
+  <div
+    class="scroll-container"
+    style="--offset-index: {currentIndex}; --milestone-count: {milestones.length};">
```
Comment (thejameswang) [inline src/components/about-us/MilestonesSection.svelte:116]: Inline?

**Review by thejameswang — APPROVED** [review]
Screen Shot 2019-08-22 at 11 18 10 AM
Largest screen has the words blocked, so that is a minor fix for later 

Screen Shot 2019-08-22 at 11 18 52 AM
The tab link is a 404, but that could be just something I don't understand

Screen Shot 2019-08-22 at 11 20 00 AM
Nit: when the site is mobile, the milestones aren't centered. 

Screen Shot 2019-08-22 at 11 21 06 AM
Nit: Feel like some of these spaces are weird

Screen Shot 2019-08-22 at 11 21 33 AM
Home still kinda F'ed 

In all, it's fine. I don't know Svelte enough to get into the nitty gritty. I poked at some stuff I saw that were off, but I'm sure you know all of them.

File: src/components/about-us/MilestonesSection.svelte (hunk 2 above)
Comment (ArkaneMoose) [reply src/components/about-us/MilestonesSection.svelte:116]: I think this is the only way to change the CSS variable based on the value of the JavaScript variable, at least until this issue is resolved: https://github.com/sveltejs/svelte/issues/758

**Review by ArkaneMoose — CHANGES_REQUESTED** [review]
Most of the code looks good to me! My only concern is that the Contentful stuff is getting a little unwieldy to work with.

As we talked about earlier, I've made a PR onto this PR to move the Contentful transform logic into the build using the Rollup plugin, thereby eliminating the need for `contentful-utils.js` entirely. See PR #27.

Once you've looked over those changes and merged them in, this PR looks ready to merge!

File: src/components/about-us/MilestonesSection.svelte (hunk 2 above)
Comment (ghost) [reply src/components/about-us/MilestonesSection.svelte:116]: Huh, didn't know they had a solution in the works for that. But you're right @ArkaneMoose

**Review by ArkaneMoose — APPROVED** [review]
LGTM!

File: src/components/about-us/HeaderSwoosh.svelte (hunk 1 above)
Comment (ghost) [reply src/components/about-us/HeaderSwoosh.svelte:20]: That's just an svg I pasted in. No svelte magic here 🤷‍♀ @thejameswang

**Pull request merged**